src/resolvers/resolver.ts

```diff
@@ -35,6 +54,53 @@ const resolvers = {
       await userExists.update({ profile: newProfile._id });
       return newProfile;
     },
+
+    async createCohort(_: any, { name, coordinator, phase }: any) {
+      const cohortExists = await Cohort.findOne({ name });
```

Comment:
Here also, remember to normalize the input that you will get before comparizon

```
const normalized = name.toLowerCase()
// then do the find here
const exists = await Cohort.exists({name: normalized})
// the exists method, return true or false which will be enough for us

if (exists) throw the error 
```

Reply:
> Here also, remember to normalize the input that you will get before comparizon
> 
> ```
> const normalized = name.toLowerCase()
> // then do the find here
> const exists = await Cohort.exists({name: normalized})
> // the exists method, return true or false which will be enough for us
> 
> if (exists) throw the error 
> ```

@feyton  your suggestion is a very good practice, but as mentioned before the suggested logic is being implemented in the task of programs and cohorts which will be rebased once merged.